src/gui/tray/UserModel.cpp

```diff
@@ -667,6 +698,16 @@ Q_INVOKABLE bool UserModel::isUserConnected(const int &id)
     return _users[id]->isConnected();
 }
 
+Q_INVOKABLE QUrl UserModel::statusIcon(const int &id)
```

Comment:
Should be `int id`. Never pass base types by reference (except if there's a really good reason like an out parameter but a) I'd favor a pointer for those and b) I try to avoid them altogether).

I know it was done like that in UserModel at other places before, but it was blatantly wrong. Honestly UserModel API is screaming for a giant cleanup.

Like I suspect avatarById (and now statusIcon) are in fact unused (AFAICT the QML goes through the roles anyway).